easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-GCC-8.3.0.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+# "make install" doesnt copy all the compiled binaries so we use the "MakeCp" easyblock
+# to be sure everything is copied and we run ./configure in prebuildopts
+
+# modified by Tom Strempel
```

Comment:
Please add your attribution to the the list from `IDBA-UD-1.1.3-GCC-8.2.0-2.31.1.eb`, the most recent easyconfig.